### PR TITLE
UI

### PR DESCRIPTION
**Pull request by bzapa**
fixed commit history

### CODE REVIEW

File: app/src/main/java/programatorus/client/screens/boards/ManageBoardsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+        binding.tabs.getTabAt(ALL)?.view?.setOnClickListener { useAll() }
+        binding.tabs.getTabAt(FAVORITES)?.view?.setOnClickListener { useFavorites() }
+
+        binding.btn.setOnClickListener {
+            Log.d(
+                "fav list:",
+                "fav ${binding.favBoards.getBoards()} \n all ${binding.allBoards.getBoards()}"
+            )
+        }
```
Comment (bgrzesik) [inline app/src/main/java/programatorus/client/screens/boards/ManageBoardsFragment.kt]: Why not `with (binding) { ... }` ?

File: app/src/main/java/programatorus/client/screens/boards/all/BoardListAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            binding.filename.text = item.name
+            binding.checkbox.isChecked = item.isSelected.get()
+
+            binding.checkbox.setOnClickListener {
+                item.toggle()
+            }
+
+            binding.root.setOnClickListener {
+                onItemClickListener(item)
+            }
```
Comment (bgrzesik) [inline app/src/main/java/programatorus/client/screens/boards/all/BoardListAdapter.kt]: Same here 😄

**Review by bgrzesik — DISMISSED** [review]

File: app/src/main/java/programatorus/client/screens/boards/ManageBoardsFragment.kt (hunk 1 above)
Comment (bzapa) [reply app/src/main/java/programatorus/client/screens/boards/ManageBoardsFragment.kt]: rewritten using `with (binding) { ... }`
removed
```
binding.favBoards.visibility = View.GONE
binding.allBoards.visibility = View.VISIBLE
``` 
as they already have correct visibility from xml file

File: app/src/main/java/programatorus/client/screens/boards/all/BoardListAdapter.kt (hunk 2 above)
Comment (bzapa) [reply app/src/main/java/programatorus/client/screens/boards/all/BoardListAdapter.kt]: rewritten using with (binding) { ... }

**Review by bgrzesik — APPROVED** [review]

**Pull request merged**